src/wait_for_deps.py

```diff
@@ -48,6 +48,11 @@ def main():
             attempt += 1
     print()
 
+    # Check PostgreSQL role service and stops immediately in case of any error
+    print(f"Checking PostgreSQL role service")
+    check_jdbc_role_service()
```

Comment:
I do not see any reason, why not set `role_service_db_uri` and `role_service_schema` as params.